src/plugins/intel_gna/src/gna_device.hpp

```diff
@@ -127,6 +128,11 @@ class GNADeviceHelper : public GNADevice {
         return allAllocations;
     }
 
+    const size_t getMemAlignment() const {
+        IE_ASSERT(m_mem_alignment);
```

Comment:
Why do we need this assertion?
Is there a chance to have 0 here?
Shouldn't be it rather done on set?

Reply:
good finding, there is no need for assertion here